examples/models/sklearn_iris/sklearn_iris.ipynb

```diff
@@ -177,25 +177,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [],
    "source": [
-    "res=!curl  -s http://localhost:8003/seldon/seldon/seldon-deployment-example/api/v0.1/predictions -H \"Content-Type: application/json\" -d '{\"data\":{\"ndarray\":[[5.964,4.006,2.081,1.031]]}}'"
```

Comment:
we should not change port `8003` to `80` - most examples assumes that default gateway is on `8003`

Reply:
Done